numpy/ma/tests/test_core.py

```diff
@@ -1351,6 +1351,17 @@ def test_minmax_dtypes(self):
             assert masked_array([-cmax, 0], mask=[0, 1]).max() == -cmax
             assert masked_array([cmax, 0], mask=[0, 1]).min() == cmax
 
+    @pytest.mark.parametrize("time_type", [np.datetime64("NaT", 's'),
+                                          np.timedelta64("NaT", 's')])
```

Comment:
Please don't use the scalars here, use the dtype or dtype string like "M8[s]".

Reply:
That's good. I don't know it before. I will follow your suggestion.